apps/client/app/goods-receipts/[id]/page.tsx

```diff
@@ -0,0 +1,29 @@
+// app/goods-receipts/[id]/page.tsx
```

Comment:
might be a leftover from ai, which would be great. makes us all more efficient :)

Reply:
Using AI to help create a lot of boilerplate code is definitely efficient especially when you are using the same pattern in multiple parts of the application